pkg/cache/clusterqueue.go

```diff
@@ -200,6 +201,7 @@ func filterQuantities(orig FlavorResourceQuantities, resourceGroups []kueue.Reso
 }
 
 func (c *ClusterQueue) updateResourceGroups(in []kueue.ResourceGroup) {
+	oldRG := c.ResourceGroups
```

Comment:
I don't understand why this matters for solving #1344

The scenario doesn't talk about changing the resource groups in the CQ.

Reply:
Ahhhh, I read it wrong. We were increasing the counter unnecessarily